ada_2_fhir-r4/zibs2020/payload/0.0/nl-core-Procedure.xsl

```diff
@@ -59,11 +67,27 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                     <basedOn>
                         <xsl:call-template name="makeReference">
                             <xsl:with-param name="in" select="$in"/>
-                            <xsl:with-param name="profile" select="'nl-core-Procedure-ServiceRequest'"/>
+                            <xsl:with-param name="profile" select="'nl-core-Procedure-request'"/>
                         </xsl:call-template>
                     </basedOn>
                 </xsl:for-each>
-                <status value="completed"/>
+                <status>
```

Comment:
Why is there logic present w.r.t. a ProcedureStartDate that lies in the future, while the nl-core-Procedure-event is only applied in case the ProcedureStartDate lies in the past?